[CN-813] Fix WAN Replication Related Docs (#103)
hazelcast/hazelcast-platform-operator-docs: docs/modules/ROOT/pages/advanced-networking.adoc

include::ROOT:example$/advanced-networking-wan.yaml[]
----

> Possible values for `serviceType` are: `ClusterIP` and `LoadBalancer`. After applying this example configuration, there will be 1 service created with type of `NodePort` which exposes 5 ports: `5710`, `5711`, `5712`, `5713`, `5714`. If you don't provide a serviceType, it will be set to `LoadBalancer` by default.

Reviewer: Maybe it is better to make the serviceType monospace.

> Possible values for `serviceType` are: `ClusterIP` and `LoadBalancer`. After applying this example configuration, there will be 1 service created with type of `NodePort` which exposes 5 ports: `5710`, `5711`, `5712`, `5713`, `5714`. If you don't provide a serviceType, it will be set to `LoadBalancer` by default.

Reviewer 2: ```suggestion
Possible values for `serviceType` are: `ClusterIP` and `LoadBalancer`. After applying this example configuration, there will be 1 service created with type of `NodePort` which exposes 5 ports: `5710`, `5711`, `5712`, `5713`, `5714`. If you don't provide a `serviceType`, it will be set to `LoadBalancer` by default.
```

> Possible values for `serviceType` are: `ClusterIP` and `LoadBalancer`. After applying this example configuration, there will be 1 service created with type of `NodePort` which exposes 5 ports: `5710`, `5711`, `5712`, `5713`, `5714`. If you don't provide a `serviceType`, it will be set to `LoadBalancer` by default.

Reviewer 3: ```suggestion
Possible values for `serviceType` are: `ClusterIP` and `LoadBalancer`. After applying this example configuration, there will be 1 service created with a configured type that exposes 5 ports: `5710`, `5711`, `5712`, `5713`, `5714`. If you don't provide a `serviceType`, it will be set to `LoadBalancer` by default.
```
NodePort is not supported, AFAIK.

Author: Replaced it with `LoadBalancer`
